app/services/proxy_api/tftp.rb

```diff
@@ -39,7 +39,7 @@ def fetch_boot_file(args)
     end
 
     # returns the TFTP boot server for this proxy
-    def bootServer
+    def boot_server
```

Comment:
I'm not sure if this is not used anywhere, it is a public API, so it definetelly could.
So in regards to https://community.theforeman.org/t/how-to-track-developer-changes/26523/5 I'd just drop a notice about this on discoures (unfortunatelly we do not have specialized section for this yet).